projects/fuzzing-ethereum-network.md

```diff
@@ -0,0 +1,107 @@
+communicate through the ethereum network and be able to start integrating my fuzzers.
+
+For the fuzzing, there are a lot of fuzzing techniques, the goal will be to identify the most appropriate to integrate them in priority and then to use the maximum of different techniques in order to have the most results.
+
+For the fuzzing test at first I will start by sending messages on a private network, the nodes will use reth. This will allow me to test the first reactions to fuzzing, and to potentially detect issues on the devp2p or issues that are specific to reth.
+
+Then once I have tested this well I plan to launch my fuzzing directly on a testnet in order to interact with other nodes, which will use other implementations of ethereum. This could make it possible to find vulnerabilities on the devp2p but also other vulns specific to various implementations of ethereum (example: a message crashes a node using geth but the same message does not cause anything on reth).
+
+## Roadmap
+
+Over the remaining 3 months of the epf, here is how I divided the work:
+
+First month :
+
+The first month will be devoted to the implementation of devp2p using reth, research on the specifications of the various devp2p protocols as well as to the development of first fuzzers.
+The goal will be to connect to the ethereum network and send messages using fuzzers.
+
+The last two month :
+
+The last two months will be devoted to the study of the results obtained by the fuzzers, in order to determine which other fuzzing techniques could be useful.
+And the development of additional fuzzers including more advanced fuzzers.
+
+## Possible challenges
+
+The first challenge will be to have a good understanding of devp2p and the messages exchanged between the nodes.
+Then to integrate and modify correctly the network part of reth to send messages.
+
+The second problem I could encounter is finding security vulns via fuzzers, using fuzzers does not necessarily imply the possibility of finding vulns and bugs, it will be up to me to develop good fuzzers and use the appropriate techniques to achieve this.
+
+In order to overcome these problems I intend to invest myself and spend time on the development of this project, I intend to dedicate between 50 and 60 hours per week to it.
+
+## Goal of the project
+
+The goal of the project is to contribute to the security aspect of ethereum by sending malicious messages on the ethereum network
+in order to find potential security vulnerabilities or bugs.
+
+The project will be considered successful if by the end of the epf he will have managed to find some.
+
+The end of the epf will not necessarily mark the end of this project, it will always be available and will always be able to receive
+contributions.
+
+Personally my goal is to contribute to the security of ethereum, so continuing to contribute might interest me in the future :)
+
+## Collaborators
+
+### Fellows 
+
+- [Mohas](https://github.com/mohasdev) 
+
+### Mentors
```

Comment:
for devp2p and eth protocol related stuff, you might want to reach out to Felix from geth